app/views/energy_tariffs/energy_tariffs/_charges_table.html.erb

```diff
@@ -1,30 +1,37 @@
 <table class="table table-charges">
   <tbody>
-  <% if energy_tariff.energy_tariff_charges.any? %>
-    <% energy_tariff.energy_tariff_charges.each do |energy_tariff_charge| %>
+    <% if energy_tariff.has_any_standing_charges? %>
+      <% if energy_tariff.energy_tariff_charges.any? %>
```

Comment:
I'm not sure this is needed here?  `energy_tariff.energy_tariff_charges.each` should just skip if empty?